packages/viz/src/lib/style/helpers/color-continuous-style.ts

```diff
@@ -0,0 +1,75 @@
+import { mix as chromaMix } from 'chroma-js';
```

Comment:
I'm gonna make a change in this file, using chroma.scale vs chroma.mix. I think it will be easier and more efficient.

Reply:
Cool! Thanks!